popup.el

```diff
@@ -150,6 +150,8 @@ untouched."
 
 (defun popup-vertical-motion (column direction)
   "A portable version of `vertical-motion'."
+  (if (functionp 'line-number-display-width)
+      (setq column (- column (line-number-display-width 'columns))))
```

Comment:
I think this should be using `when` macro and instead of checking the function defined, should check with if `display-line-numbers-mode` is enabled or not. Overall should look like this.

```el
(when display-line-numbers-mode
  (setq column (- column (line-number-display-width 'columns))))
```